content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,173 @@
 
-.. _discuss_app/notification_preferences:
+Inbox, starred, and history
+===========================
 
-Choose notifications preference
-===============================
+Upon opening the :menuselection:`Discuss app`, the :guilabel:`Discuss` dashboard appears.
 
-Access user-specific preferences for the *Discuss* app by navigating to :menuselection:`Settings app
---> Users --> User --> Preferences tab`.
+On the :guilabel:`Discuss` dashboard, unread messages are visible in the :guilabel:`Inbox`.
+:guilabel:`Starred` is where starred messages are stored. :guilabel:`History` shows *Chatter*
+updates for records in the Odoo database the user has been assigned to, or tagged on.
 
-.. image:: discuss/preferences-user.png
-   :alt: View of the Preferences tab for Odoo Discuss.
+Direct messages
+===============
 
-By default, the :guilabel:`Notification` field is set as :guilabel:`Handle by Emails`. With this
-setting enabled, a notification email will be sent by Odoo every time a message is sent from the
-chatter, a note is sent with an `@` mention (from chatter), or a notification is sent for a record
-that the user follows. Something that triggers a notification is changing of the stage (if an email\
-is configured to be sent, for example if the task is set to :guilabel:`Done`).
+*Direct messages* allow the user to communicate privately with one or multiple team members. To
+start a new direct message, click the :icon:`fa-plus` icon, next to :guilabel:`Direct Messages` on
+the :guilabel:`Discuss` dashboard, and enter the name of the desired person in the :guilabel:`Start
+a conversation` search bar that appears.
 
-By choosing :guilabel:`Handle in Odoo`, the above notifications are shown in the *Discuss* app's
-*inbox*. Messages can have the following actions taken on them: respond with an emoji by clicking
-:guilabel:`Add a Reaction`, or reply to the message by clicking on :guilabel:`Reply`. Additional
-actions may include starring the message by clicking :guilabel:`Marked as Todo`, or pinning the
-message by selecting :guilabel:`Pin` or even mark the message as unread by selecting
-:guilabel:`Marked as unread`.
+.. tip::
+   Multiple names can be selected in the :guilabel:`Start a conversation` search bar. Once all of
+   the names have been entered, press :kbd:`Enter`.
 
-.. image:: discuss/reactions-discuss.png
-   :alt: View of an inbox message and its action options in Odoo Discuss.
+Direct message actions
+----------------------
 
-Clicking :guilabel:`Mark as Todo` on a message causes it to appear on the :guilabel:`Starred` page,
-while clicking :guilabel:`Mark as Read` moves the message to :guilabel:`History`.
+Hover over a direct message in the chat window to see a menu of actions to take on the message.
 
-.. image:: discuss/starred-messages.png
-   :alt: View of messages marked as todo in Odoo Discuss.
+.. image:: discuss/direct-message-actions.png
+   :alt: View direct message actions.
```

Comment:
Move image below text, and replace image– we are already using icons in the written instruction, there is little need to show only the icons again in an image. Instead, I recommend a screenshot of the entire header :)


<img width="467" alt="conversation-actions" src="https://github.com/user-attachments/assets/1039ce16-d21b-4e06-9902-0b40e7ebdf39">

```suggestion
```